src/snowflake/snowpark/_internal/analyzer/snowflake_plan.py

```diff
@@ -678,7 +678,6 @@ def large_local_relation_plan(
             if thread_safe_session_enabled
             else random_name_for_temp_object(TempObjectType.TABLE)
         )
-        temp_table_name = f"temp_name_placeholder_{generate_random_alphanumeric()}"
```

Comment:
I forgot to do clean-up correctly when I added param protection for thread-safe changes again. See line 677 where this is already initialized correclty.